### PR TITLE
[xdoctest] reformat example code with google style in No 101

### PR DESCRIPTION
**Pull request by Liyulingyue**
### PR types

Others 
### PR changes

Others 
### Description

https://github.com/PaddlePaddle/Paddle/issues/55629

### CODE REVIEW

File: python/paddle/nn/utils/weight_norm_hook.py [hunk 1]
```diff
@@ -219,21 +219,20 @@ def remove_weight_norm(layer, name='weight'):
...
+            >>> paddle.seed(2023)
+
+            >>> conv = Conv2D(3, 5, 3)
+            >>> wn = weight_norm(conv)
+            >>> print(conv.weight_g)
+            Parameter containing:
+            Tensor(shape=[5], dtype=float32, place=Place(cpu), stop_gradient=False,
+                   [1.35883713, 1.32126212, 1.56303072, 1.20874095, 1.22893476])
+            >>> remove_weight_norm(conv)
             # The following is the effect after removing the weight norm:
```
Comment (Liyulingyue) [inline python/paddle/nn/utils/weight_norm_hook.py]:
```suggestion
            >>> # The following is the effect after removing the weight norm:
```

File: python/paddle/nn/utils/weight_norm_hook.py [hunk 2]
```diff
@@ -219,21 +219,20 @@ def remove_weight_norm(layer, name='weight'):
...
-            # print(conv.weight_g)
+            >>> print(conv.weight_g)
             # AttributeError: 'Conv2D' object has no attribute 'weight_g'
```
Comment (Liyulingyue) [inline python/paddle/nn/utils/weight_norm_hook.py]:
```suggestion
            >>> # AttributeError: 'Conv2D' object has no attribute 'weight_g'
```

File: python/paddle/nn/utils/weight_norm_hook.py [hunk 3]
```diff
@@ -219,22 +219,21 @@ def remove_weight_norm(layer, name='weight'):
...
+
+            >>> conv = Conv2D(3, 5, 3)
+            >>> wn = weight_norm(conv)
+            >>> print(conv.weight_g)
+            Parameter containing:
+            Tensor(shape=[5], dtype=float32, place=Place(cpu), stop_gradient=False,
+                   [1.35883713, 1.32126212, 1.56303072, 1.20874095, 1.22893476])
+            >>> remove_weight_norm(conv)
+            >>> # The following is the effect after removing the weight norm:
+            >>> print(conv.weight_g)
```
Comment (SigureMo) [inline python/paddle/nn/utils/weight_norm_hook.py]:
```suggestion
            >>> # print(conv.weight_g)
```

注释都说了会报错，为什么还要解开注释呢？

**Review by SigureMo — APPROVED** [review]
LGTMeow 🐾

**Pull request merged**